soc/xtensa/intel_s1000/linker.ld

```diff
@@ -368,6 +368,11 @@ SECTIONS
     *(.rodata.*)
     *(.gnu.linkonce.r.*)
     *(.rodata1)
+
+    . = ALIGN(4);
```

Comment:
Same thing here.  If the alignment isn't required, please remove.

Reply:
Done